DEPS

```diff
@@ -43,12 +45,12 @@ vars = {
   'checkout_oculus_sdk' : False,
   'checkout_openxr' : False,
   'mac_xcode_version': 'default',
+
+  'react_native_target':'default'
 }
 
 deps = {
-  # 'src/react-native'                      : Var('react_native_git') + '@' + Var('react_native_revision'),
-  # react-native patch to fix cxx textlayoutmanager build break
-  'src/react-native'                      : 'https://github.com/Kudo/react-native.git' + '@' + 'deb66012fe550d536420931e47f5573e798e048a',
+  'src/react-native'                      : Var('react_native_'+Var('react_native_target')+'_git') + '@' + Var('react_native_'+Var('react_native_target')+'_revision'),
```

Comment:
nested `Var` is not very clear. is it possible to rewrite this? even to have additional variables, that would be better.

Reply:
yes sure. can define additional var to generate the final string . will update it.